parser/indexedfile.go

```diff
@@ -155,7 +155,7 @@ func indexFiles(files []string, indexingThreads int, res *resources.Resources) [
 						"error": err.Error(),
 					}).Debug("An error was encountered while indexing a file")
 					//errored on files will be nil
-					fmt.Printf("\t[!] An error occured while indexing %v. Perhaps this log is empty?", files[j])
```

Comment:
This is an unrelated change that caused the linter to fail. I fixed the spelling but the error will go away when/if https://github.com/activecm/rita/pull/563 is pulled in as well.

Reply:
pulled into master and changed merged into this branch.